frontend/src/constants/policySummary.js

```diff
@@ -14,7 +14,7 @@ export const POLICY_SUMMARY = (
     - [필수] 이름, 이메일, 연락처, 성별, 생년월일
     <br />
     <br />
-    3. 개인정보 보유 및 이용기간
+    3. 개인정보 보유 및 이용 기간
```

Comment:
회사에서 받은 문구라 그대로 두는 게 좋을 것 같아요.

```suggestion
    3. 개인정보 보유 및 이용기간
```